### PR TITLE
Attempt 2

### PR DESCRIPTION
**Pull request by Kyle-Westberg**

### CODE REVIEW

File: threef.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package main
+
+import ( 
+    "fmt"
+    "os"
+    "strconv"
+) 
+
+func main() {
```
Comment (mcdowellm) [inline threef.go:9]: Looks good to me.

**Comment by lorissaparks** [comment]
some files have a warning about no new line at end of file. Not exactly sure what that means or how important it is

**Comment by lorissaparks** [comment]
Everything else looks good.

File: Makefile [hunk 2]
```diff
@@ -5,5 +5,5 @@ part1: threef.go
 	go build -o part1 threef.go
 part2: main.c rectangle.c rectangle.h
 	c99 -Wall -Werror main.c rectangle.c -o part2
-test:
```
Comment (mcdowellm) [inline Makefile]: Yep correct commit for Makefile

**Pull request merged**